dashboard/consts.py

```diff
@@ -9,6 +11,13 @@
 ORGANIZE_DATA_INTERVAL_SECONDS = 2
 REDIS_KEY_DASHBOARD_RPC = "dashboard_rpc"
 REDIS_KEY_GCS_SERVER_ADDRESS = "GcsServerAddress"
+# GCS check alive
+GCS_CHECK_ALIVE_MAX_COUNT_OF_RPC_ERROR = env_integer(
+    "GCS_CHECK_ALIVE_MAX_COUNT_OF_RPC_ERROR", 10)
+GCS_CHECK_ALIVE_INTERVAL_SECONDS = env_integer(
+    "GCS_CHECK_ALIVE_INTERVAL_SECONDS", 5)
+GCS_RETRY_CONNECT_INTERVAL_SECONDS = env_integer(
+    "GCS_RETRY_CONNECT_INTERVAL_SECONDS", 2)
```

Comment:
Sorry, where is this used?

Reply:
Sorry, I have renamed the `CONNECT_GCS_INTERVAL_SECONDS` to `GCS_RETRY_CONNECT_INTERVAL_SECONDS` to make it more clear. I will fix the code.